src/app/pages/CpsMap/index.jsx

```diff
@@ -48,6 +48,7 @@ const CpsMapContainer = ({ pageData }) => {
   );
   const firstPublished = getFirstPublished(pageData);
   const lastPublished = getLastPublished(pageData);
+  const aboutTags = path(['metadata', 'tags', 'about'], pageData);
```

Comment:
Can this be moved into #lib/utilities/parseAssetData (like `getFirstPublished` and `getLastPublished` above?) Or is the path different for an article vs a CPS asset?

Reply:
parse in parseAssetData is maybe the wrong verb here

Reply:
Yeah I think you're right. Should we add a tech_debt issue to address it later?